src/operators/timeoutWith.ts

```diff
@@ -43,12 +43,12 @@ class TimeoutWithSubscriber<T, R> extends OuterSubscriber<T, R> {
     return this._hasCompleted;
   }
 
-  constructor(destination: Subscriber<T>,
+  constructor(public destination: Subscriber<T>,
               private absoluteTimeout: boolean,
               private waitFor: number,
               private withObservable: Observable<any>,
               private scheduler: Scheduler) {
-    super(destination);
```

Comment:
I know I've seen this pattern, but I'm pretty sure it's relegated to "inner subscribers". If not, we need to have a look at those other operators.